### ISSUE TITLE
Footnotes from table get displayed twice in LaTeX

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1522]
A minimal test case:

``` restructuredtext
====== ======
header header
====== ======
[#f1]_ [#f2]_
====== ======

.. [#f1] footnote 1
.. [#f2] footnote 2
```

When built with Sphinx LaTeX builder and compiled to PDF, the result looks like:

| header | header |
| --- | --- |
| [^3](footnote1) | [^4](footnote2) |

///Footnotes Go Here///

The generated `.tex` file has every footnote once, so I suppose it is a problem with one of Sphinx's `.sty` files.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1522
- Originally reported by: Dmitry Shachnev
- Originally created at: 2014-07-26T19:48:08.790

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-26 13:38:13+00:00_

Fix: footnotes from table get displayed twice in LaTeX. To fix this, restoring footnote support hotfix for tabulary.sty that had been lost at f938a95 for #949. Closes #1522

→ <<cset e9aae6563d645f56819e412481b40d21c3a7ba70>>

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-26 13:54:06+00:00_

This problem was caused by updating `tabulary.sty` from version `2007/10/02 v0.9` to version `2008/12/01 v0.9` at f938a95 for #949. The old tabulary.sty that is bundled in the Sphinx contains a patch to support footnote in the table (I think it was brought from https://groups.google.com/d/msg/comp.text.tex/YDOlFEuzBQk/b-rZ0Pbece8J), but the patch was lost by the update.

issue #949 suggests fixing a width bug that described at http://www.latex-community.org/forum/viewtopic.php?f=5&t=3238. However I think the patch to support footnote in the table is not necessary to be deleted.
